src/main/java/com/stuypulse/robot/commands/shooter/ExtendHoodCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.stuypulse.robot.commands.shooter;
+
+import com.stuypulse.robot.subsystems.Shooter;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+public class ExtendHoodCommand extends InstantCommand {
```

Comment:
These class names are TECHNICALLY missing Shooter, but we will let you know if you need to change them before we merge (so dwai yet)

Reply:
change the name. it should start with Shooter